.pre-commit-config.yaml

```diff
@@ -0,0 +1,119 @@
+exclude: Pipfile\.lock|migrations|\.idea|node_modules
+
+repos:
+  - repo: local # More local hooks are defined at the bottom.
+    hooks:
+      - id: i18n
+        name: i18n
+        entry: bash -c 'if [ ! -f "$(dirname "$dir")"/src/locales/ar.json ]; then pnpm i18n; fi'
+        language: system
+        pass_filenames: false
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
+      - id: check-executables-have-shebangs
+      - id: check-json
+        exclude: tsconfig.json # contains comments
+      - id: check-case-conflict
+      - id: check-toml
+      - id: check-merge-conflict
+      - id: check-xml
+      - id: check-yaml
+      - id: end-of-file-fixer
+        exclude: test/tapes/.+\.json5
+      - id: check-symlinks
+      - id: mixed-line-ending
+      - id: fix-encoding-pragma
+        args:
+          - --remove
+      - id: requirements-txt-fixer
+
+  - repo: https://github.com/PyCQA/isort
```

Comment:
Why do we need the Python-specific configs here (even if they are noop)? For the future monorepo support?

Reply:
The goal (in the next step) is to sync this config from WordPress/openverse. So I just left it as is.